hotsos/core/host_helpers/cli.py

```diff
@@ -228,7 +234,15 @@ def __call__(self, *args, **kwargs):
         elif self.json_decode:
             output = json.load(open(self.path))
         else:
-            output = open(self.path, 'r').readlines()
+            output = []
+            ln = 0
+            for line in open(self.path, 'rb'):
```

Comment:
I know pylint will usually complain about not using a "with open()" since that causes the file to get closed on leaving the branch. Could probably fix this with something like:

with open(self.path, 'rb') as f:
    for line in f.readlines():
        ...

Reply:
+1 will fix